packages/vue-renderer/src/spa-meta.js

```diff
@@ -30,8 +30,9 @@ export default class SPAMetaRenderer {
     return vm.$meta().inject()
   }
 
-  async render({ url = '/', req = {} }) {
-    const cacheKey = `${req.modernMode ? 'modern:' : 'legacy:'}${url}`
+  async render({ url = '/', req = {}, _generate }) {
```

Comment:
This is extracting `renderRoute` `context`. Are you sure this context also includes `_generate`? I think it should be accessed with `this.options._generate`

Reply:
I found this flag from https://github.com/nuxt/nuxt.js/blob/dev/packages/generator/src/generator.js#L204

The reason why I did't use `this.options._generate` is: `generate` in `build` or programmatic `generate` don't have `_generate`

Reply:
Nice find 😊 I think that's an older flag but you are right